index.js

```diff
@@ -361,6 +363,15 @@ module.exports = fp(async function (app, opts) {
     )
 
     if (execution.errors) {
+      execution.errors = execution.errors.map(e => {
+        if (e.originalError instanceof FastifyGraphQLError) {
```

Comment:
I would prefer if we duck-typed this instead of checking for inheritance, so any error with `code` and `additionalProperties`   would work.